.pre-commit-config.yaml

```diff
@@ -40,32 +40,32 @@ repos:
         additional_dependencies: ["flake8-bugbear==20.1.4"]
         args: ["--select=B,E9,F63,F7,F82"]
 
-  - repo: https://github.com/returntocorp/semgrep
-    rev: 'v0.12.0'
-    hooks:
-      - id: semgrep-develop
-        name: Semgrep Python
-        types: [python]
-        exclude: "^semgrep/tests/.+$|^install-scripts/.+$|^release-scripts/.+$|^semgrep/setup.py$"
-        args: ['--config', 'https://semgrep.live/p/python', '--error']
+  # - repo: https://github.com/returntocorp/semgrep
```

Comment:
I assume you don't want to comment changes to this file?

Reply:
Oh yeah, we just changed this so that it would run on my machine, right? Or should I delete the commented-out pieces and commit the change? I guess I didn't understand what the impact of this was/if it was supposed to be pushed up